pkg/proxyserver/getter/loggetter.go

```diff
@@ -0,0 +1,65 @@
+package getter
+
+import (
+	"time"
+
+	utilnet "k8s.io/apimachinery/pkg/util/net"
+	"k8s.io/client-go/dynamic"
+	restclient "k8s.io/client-go/rest"
+)
+
+// ClientConfig is to define the configuration to connect klusterlet
+type ClientConfig struct {
```

Comment:
I did not see any place that uses this `ClientConfig`

Reply:
deleted